docs/modules/usage/llms/groq.md

```diff
@@ -0,0 +1,19 @@
+# Running LLMs on Groq
+
+OpenHands uses LiteLLM to make calls to chat models on Groq. You can find their full documentation on using Groq as provider [here](https://docs.litellm.ai/docs/providers/groq). When running OpenHands, it's simpler: you only need to set the options below.
```

Comment:
Can you move the last sentence to the Configuration section please so it lines up with how we do other ones?

"When running OpenHands, you'll need to set the following in the OpenHands UI through the Settings:
...
"

Reply:
I adjusted it, still kept a little before. I just wanted to suggest somehow that the litellm docs we just linked to may be useful, but not everything in them is necessary, we literally make it simpler (e.g. no need for env vars of specific names)

Reply:
Actually, I changed my mind and removed that extra bit. It lines up, yes.